packages/toaster/components/Toast.tsx

```diff
@@ -26,21 +26,21 @@ export interface ToastProps {
   title: React.ReactElement<HTMLElement> | string;
   description?: React.ReactElement<HTMLElement> | string;
   id: ToastId;
-  appearance?: "default" | "success" | "warning" | "danger";
+  appearance: "default" | "success" | "warning" | "danger";
   autodismiss?: boolean;
   onDismiss?: (event?: React.SyntheticEvent<HTMLElement>) => void;
   dismissToast?: (dismissedToastId: ToastId) => void;
   primaryAction?: React.ReactElement<HTMLElement>;
   secondaryAction?: React.ReactElement<HTMLElement>;
 }
 
-const SuccessIcon = props => (
+const SuccessIcon = (
```

Comment:
We could replace these with the Icon component. Toast components were created before we had the Icon component.

Reply:
i wanted to do that but storybook was not open at that moment - so i just compared the svg-paths which did not match up and abandoned the idea.

thanks a lot for paying close attention here! 😍